### PR TITLE
Gsuite download detection

### PR DESCRIPTION
**Pull request by calkim-panther**
### Background

Gsuite high download count detection
Gsuite high delete count detection

### Testing

* <Testing steps>
```
GSuite.Drive.Many.Documents.Deleted
	[PASS] Test-5c89ab
		[PASS] [rule] true
		[PASS] [title] GSuite: [homer.simpson@simpsons.com] has deleted [100] documents from Google Drive.
		[PASS] [dedup] GSuite: [homer.simpson@simpsons.com] has deleted [100] documents from Google Drive.
		[PASS] [alertContext] {"delete_count": 100, "deleted_files": ["importantdoc1", "importantdoc2", "importantdoc100"], "user": "homer.simpson@simpsons.com"}

Google.Drive.High.Download.Count
	[PASS] Result Row
		[PASS] [rule] true
		[PASS] [title] GSuite: [homer.simpson@simpsons.com] downloaded [23] files from Google Drive.
		[PASS] [dedup] GSuite: [homer.simpson@simpsons.com] downloaded [23] files from Google Drive.
		[PASS] [alertContext] {"download_count": 23, "downloaded_files": ["all_hands01.mov", "all_hands02.mov", "all_hands03.mov", "all_hands23.mov"], "user": "homer.simpson@simpsons.com"}

```

### CODE REVIEW

File: queries/gsuite_queries/GSuite_Many_Docs_Downloaded_Query.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Query: |
+    SELECT
+        actor:email AS user,
+        ARRAY_AGG( DISTINCT parameters:doc_title) AS downloaded_files,
+        ARRAY_SIZE(downloaded_files) as download_count
+    FROM gsuite_activityevent
+    WHERE p_occurs_since('1 day')
+        AND name = 'download'
+    GROUP BY actor:email
+    HAVING download_count > 10
```
Comment (edyesed) [inline queries/gsuite_queries/GSuite_Many_Docs_Downloaded_Query.yml:13]: I wonder if we should time_slice based on the hour

**Review by edyesed — DISMISSED** [review]
adds two detections based on queries for 
* User downloading `>10` documents from google drive in one day
* User deleting `>10` documents from google drive in one day.

File: queries/gsuite_queries/GSuite_Many_Docs_Downloaded_Query.yml (hunk 1 above)
Comment (calkim-panther) [reply queries/gsuite_queries/GSuite_Many_Docs_Downloaded_Query.yml:14]: added time_slicing

File: queries/gsuite_queries/GSuite_Many_Docs_Deleted_Query.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Query: |
+    SELECT
+        actor:email AS user,
+        ARRAY_AGG( DISTINCT parameters:doc_title) AS deleted_files,
+        ARRAY_SIZE(deleted_files) as delete_count
+    FROM gsuite_activityevent
+    WHERE p_occurs_since('1 day')
+        AND name = 'trash'
+    GROUP BY actor:email
+    HAVING delete_count > 10
```
[the remark this answers is not in the document]
Comment (calkim-panther) [reply queries/gsuite_queries/GSuite_Many_Docs_Deleted_Query.yml:14]: added time_slice

**Review by edyesed — APPROVED** [review]
adds detections for tunable, mass gsuite downloads and deletes

**Pull request merged**